src/index.js

```diff
@@ -0,0 +1,157 @@
+    tempValue.style.color = state.defaultColor;
+    tempValue.textContent = 0;
+    sky.textContent = "";
+};
+
+const registerEventHandlers = () => {
+    state.increaseTempButton.addEventListener("click", increaseTemp, changeColor, changeLandscape);
+    state.decreaseTempButton.addEventListener("click", decreaseTemp, changeColor, changeLandscape);
+    state.currentWeatherButton.addEventListener("click", displayCurrentWeather);
+    state.cityNameInput.addEventListener("input", displayCityName); 
+    state.resetButton.addEventListener("click", setDefaultValues);
+    state.skySelect.addEventListener("change", changeSky);
+};
+
+const loadControls = () => {
+    state.increaseTempButton = document.getElementById("increaseTempControl");
+    state.decreaseTempButton = document.getElementById("decreaseTempControl");
+    state.cityNameInput = document.getElementById("cityNameInput");
+    state.cityNameOutput = document.getElementById("headerCityName");
+    state.currentWeatherButton = document.getElementById("currentTempButton");
+    state.tempValue = document.getElementById("tempValue"); 
+    state.landscape = document.getElementById("landscape");
+    state.resetButton = document.getElementById("cityNameReset");
+    state.skySelect = document.getElementById("skySelect");
+    state.sky = document.getElementById("sky");
+};
+
+const increaseTemp = () => {
+    state.tempDeg += 1;
+    tempValue.textContent = `${state.tempDeg}`
+    changeColor();
+    changeLandscape();
+};
+
+const decreaseTemp = () => {
+    state.tempDeg -= 1;
+    tempValue.textContent = `${state.tempDeg}`;
+    changeColor();
+    changeLandscape();
+};
+
+const changeColor = () => {
+    let currentTemp = tempValue.textContent;
+    if (currentTemp >= 80) {
+        tempValue.style.color = "red";
+    } else if (currentTemp >= 70) {
+        tempValue.style.color = "orange";
+    } else if (currentTemp >= 60) {
+        tempValue.style.color = "yellow";
+    } else if (currentTemp >= 50) {
+        tempValue.style.color = "green";
+    } else {
+        tempValue.style.color = "teal";
+    }
+};
+
+const changeLandscape = () => {
+    let currentTemp = tempValue.textContent;
+    if (currentTemp >= 80) {
+        landscape.textContent = `${state.hotLandscape}`;
+    } else if (currentTemp >= 70) {
+        landscape.textContent = `${state.niceLandscape}`;
+    } else if (currentTemp >= 60) {
+        landscape.textContent = `${state.midLandscape}`;
+    } else {
+        landscape.textContent = `${state.defaultLandscape}`;
+    }
+};
+
+const displayCityName = (event) => {
+    const cityInput = event.target.value; 
+    state.cityNameOutput.textContent = cityInput;
+};
+
+const changeSky = () => {
+    sky.textContent = `${skySelect.options[skySelect.selectedIndex].text}`;
+}
+
+const displayCurrentWeather = () => {
+    findLatAndLon();
+}
+
+const findLatAndLon= (query) => {
+    let latitude, longitude;
```

Comment:
What if you initialized these variables in your state object? Then on lines 118 and 119, you could do:
```
state.latitude = response.data[0].lat;
state.longitude = response.data[0].lon;